dev/com.ibm.ws.openapi.ui/swagger-ui/dist/index.html

```diff
@@ -5,7 +5,8 @@
     <title>Swagger UI</title>
     <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
     <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
-    <script defer src="liberty-swagger-ui-bundle-f46b8f492bc9c6c9d41c.js"></script><link href="liberty-swagger-ui-a44cef3790da8bb191bd.css" rel="stylesheet">
+    <script defer src="liberty-swagger-ui-bundle-f46b8f492bc9c6c9d41c.js" charset="UTF-8"></script>
+    <link href="liberty-swagger-ui-a44cef3790da8bb191bd.css" rel="stylesheet">
```

Comment:
These files are generated by the NPM build. You can't edit them by hand.

Reply:
Have removed forcesetting here - would need to see if there was anything in the webpack steps where we could add for extra security as while potentially doable elsewhere it would cause a difference between `/dist` contents and what the server was seeing